mgr/manager.go

```diff
@@ -132,20 +137,23 @@ func (cert *CertServiceManager) RenewPKI() error {
 // CheckCA checks the CA on the certificate and restarts the service
 // if needed.
 func (spec *CertServiceManager) CheckCA() error {
-	if changed, err := spec.CA.Refresh(); err != nil {
+	var err error
+	var changed bool
+	if changed, err = spec.CA.Refresh(); err != nil {
 		metrics.ActionFailure.WithLabelValues(spec.Spec.Path, "CA").Inc()
 		return err
 	} else if changed {
```

Comment:
nit: The `else` becomes useless when returning; could just drop down and `if changed { ...}`.

Reply:
ack'd, changing.

Reply:
Actually, else if here isn't hurting anything and is clear about scope- I'd prefer it be left this way.